Triangulation_2/examples/Triangulation_2/triangulation_prog1.cpp

```diff
@@ -1,27 +1,106 @@
-#include <fstream>
-
 #include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
-#include <CGAL/Triangulation_2.h>
+#include <iostream>
```

Comment:
why did you modified this example?

Reply:
I was using it as main program while getting familiar with CGAL and I continued using it to run the new implemented functions to be honest. I intended to discard these changes later and recover the original file. I know this is messy and I am sorry for that. I will now make a separate new example file in the PMP examples folder.